inspirehep/modules/disambiguation/core/ml/models.py

```diff
@@ -25,14 +25,40 @@
 from __future__ import absolute_import, division, print_function
 
 import csv
+import json
 import pickle
+from scipy.special import expit
 
 import numpy as np
+from sklearn.ensemble import RandomForestClassifier
 from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import FeatureUnion
 from sklearn.pipeline import Pipeline
 from sklearn.svm import LinearSVC
 
+from beard.similarity import CosineSimilarity
+from beard.similarity import ElementMultiplication
+from beard.similarity import EstimatorTransformer
+from beard.similarity import PairTransformer
+from beard.similarity import StringDistance
+from beard.utils import FuncTransformer
 from beard.utils import normalize_name
+from beard.utils import Shaper
+
+from inspirehep.modules.disambiguation.core.ml.utils import get_author_full_name
+from inspirehep.modules.disambiguation.core.ml.utils import get_author_other_names
+from inspirehep.modules.disambiguation.core.ml.utils import get_author_affiliation
+from inspirehep.modules.disambiguation.core.ml.utils import get_first_given_name
+from inspirehep.modules.disambiguation.core.ml.utils import get_second_given_name
+from inspirehep.modules.disambiguation.core.ml.utils import get_second_initial
+from inspirehep.modules.disambiguation.core.ml.utils import get_title
+from inspirehep.modules.disambiguation.core.ml.utils import get_abstract
+from inspirehep.modules.disambiguation.core.ml.utils import get_coauthors_from_range
+from inspirehep.modules.disambiguation.core.ml.utils import get_keywords
+from inspirehep.modules.disambiguation.core.ml.utils import get_collaborations
+from inspirehep.modules.disambiguation.core.ml.utils import get_topics
+from inspirehep.modules.disambiguation.core.ml.utils import group_by_signature
+from inspirehep.modules.disambiguation.core.ml.utils import load_signatures
```

Comment:
https://github.com/inspirehep/inspire-next/pull/3455/files#r194333002 applies as well in this file.